module3/homework/inner_product/arithmeticAverage.cpp

```diff
@@ -0,0 +1,20 @@
+#include "arithmeticAverage.hpp"
+#include <cmath>
+#include <functional>
+#include <numeric>
+
```

Comment:
I suggest you to use const iterators whenever you can :p

Reply:
+1